docs/cudf/source/conf.py

```diff
@@ -516,6 +523,17 @@ def on_missing_reference(app, env, node, contnode):
         # generates. Adding those would clutter the Sphinx output.
         return contnode
 
+    if node["refdomain"] in ("py",) and reftarget is not None:
```

Comment:
```suggestion
    if node["refdomain"] == "py" and reftarget is not None:
```
? (Unless we plan to add to this tuple later)

Reply:
I initially did this because I was worried there might be other instances of weird domains showing up (like "python") but I didn't find any. I think Breathe generates domains (and everything else) less consistently which is why we need more logic for C++ symbols below. I can change this.